OSDOCS-11520:adds support note to OVN-K migration procedure (#79795)
openshift/openshift-docs: networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc

As a cluster administrator, you can migrate to the OVN-Kubernetes network plugin from the OpenShift SDN network plugin using the _offline_ migration method or the limited live migration method.

[WARNING]
====

> There is no supported migration from OpenShift SDN network plugin to OVN-Kubernetes network plugin in {product-title} 4.17. You must migrate to the OVN-Kubernetes plugin before upgrading to {product-title} 4.17.

Reviewer: ```suggestion
No supported method exists for migrating from the OpenShift SDN network plugin to the OVN-Kubernetes network plugin in {product-title} 4.17. You must migrate to the OVN-Kubernetes plugin before upgrading to {product-title} 4.17.
```

> There is no supported migration from OpenShift SDN network plugin to OVN-Kubernetes network plugin in {product-title} 4.17. You must migrate to the OVN-Kubernetes plugin before upgrading to {product-title} 4.17.

Reviewer 2: I don't think this is the right warning; it suggests that SDN users might be able to upgrade to 4.17, but then wouldn't be able migrate to OVN-Kubernetes (at least, not in a "supported" way). But the actual problem is that SDN users can't even upgrade to 4.17 (and so they can't even get to the point where they might try "migrating from SDN to OVN in 4.17").

```suggestion
It is not possible to upgrade a cluster to {product-title} 4.17 if it is using the OpenShift SDN network plugin. You must migrate to the OVN-Kubernetes plugin before upgrading to {product-title} 4.17.
```

If we put a warning here, it should be along the lines of "it's not possible to upgrade to 4.17 if you are using SDN, so you need to migrate to OVN-K before then".
